test/e2e-test-application/e2e/tests/1-angular/i18n-context-product-app-switcher.spec.js

```diff
@@ -315,14 +315,14 @@ describe('AppSwitcher', () => {
 
       // check the title and subtlitle
       cy.get('[data-testid="luigi-topnav-title"]').should('contain', 'Application Two');
-      cy.get('.fd-shellbar__subtitle').should('contain', 'the second app');
```

Comment:
Is there a specific reason why you changed this?
If not I would revert this, because we will get failing compatibility tests.

Reply:
will revert